features/projects/components/team-members-section.tsx

```
@@ -1,132 +1,216 @@
+                    onClick={() => handleAddEmployee(employee.id!)}
+                  >
+                    {addEmployee.isPending ? (
+                      <Loader2 className="mr-2 h-4 w-4 animate-spin" />
+                    ) : (
+                      <Plus className="mr-2 h-4 w-4" />
+                    )}
+                    Add
+                  </Button>
+                </div>
+              ))
+            )}
```

ESLint

[error] Nested ternary expression should be parenthesized.

(unicorn/no-nested-ternary)